frontend/src/components/common/Resource/CreateButton.tsx

```diff
@@ -36,6 +36,7 @@ export default function CreateButton(props: CreateButtonProps) {
   const dispatchCreateEvent = useEventCallback(HeadlampEventType.CREATE_RESOURCE);
   const clusters = useClusterGroup();
   const [targetCluster, setTargetCluster] = React.useState(clusters[0] || '');
+  const [currentItem, setCurrentItem] = React.useState({}); // eslint-disable-line no-unused-vars
```

Comment:
I don't see the setCurrentItem being used here. Is this why you disabled the eslint? You could work around this by not declaring the setCurrentItem. But if that's the case, is there a need for using the state at all? Could you use a {} instead?

Reply:
changed this to a plain useRef and works perfectly